VERSION

```diff
@@ -1 +1 @@
-0.1.20
+0.1.21
```

Comment:
maybe we should consider more final versioning. As this extends API I would extend bump of minor version. But I agree that according to semver everything in version 0 basically does not follow versioning.